src/strategies/swarm-stacking/index.ts

```diff
@@ -0,0 +1,88 @@
+import { BigNumber } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { subgraphRequest } from '../../utils';
+import { getAddress } from '@ethersproject/address';
+
+export const author = 'swarm';
```

Comment:
```suggestion
export const author = 'SwarmMarkets';
```

Reply:
@ChaituVR yep good point, I've updated it rn